tests/test_s3.py

```diff
@@ -114,6 +115,107 @@ def test_metrics(self):
             'AWS/S3.NumberOfObjects.Average' in resources[0]['c7n.metrics'])
 
 
+class AccessFilter(BaseTest):
+
+    recording = os.environ.get('C7N_RECORD') == '1'
```

Comment:
Can you live with 82a84d89223bd0096f1e9c35318e41cf9e0225aa on this PR? Alternative would be to drop back to previous pattern w/ commenting out lines.